src/pyOpenMS/pyopenms/dataframes.py

```diff
@@ -304,6 +304,61 @@ def get_df(self, long : bool = False):
 
         return pd.DataFrame(data=((spec.getRT(), *spec.get_peaks()) for spec in self), columns=cols)
 
+    def get_massql_df(self):
+        """Exports data from MSExperiment to pandas DataFrames to be used with MassQL.
+
+        The Python module massql allows queries in mass spectrometry data (MS1 and MS2
+        data frames) in a SQL like fashion (https://github.com/mwang87/MassQueryLanguage).
+        
+        Returns:
+        ms1_df (pandas.DataFrame): peak data of MS1 spectra
+        ms2_df (pandas.DataFrame): peak data of MS2 spectra with precursor information
+        """
+        self.updateRanges()
+
+        def get_polarity(spectrum):
+            polarity = spectrum.getInstrumentSettings().getPolarity()
+            if polarity == IonSource.Polarity.POLNULL:
+                return 0
+            elif polarity == IonSource.Polarity.POSITIVE:
+                return 1
+            elif polarity == IonSource.Polarity.NEGATIVE:
+                return 2
```

Comment:
could we use a python enum / would this make sense?

Reply:
otherwise I might prefer -1, 0, +1

Reply:
That was my first solution as well, since it is the most intuitive. However, the massql exports positive as 1 and negative as 2, that is the reason for this choice.